### ISSUE TITLE
[QUESTION] What is encrypted and what is not in this app?

### ISSUE THREAD

**Issue by GoetheG** [RocketChat/Rocket.Chat.iOS#2473]
Hi there,

I am a non-coder and I subscribed for a Rocket.Chat service from a German provider. We chose Rocket.Chat as a secure solution for internal communication. Unfortunately, I do not understand what's encrypted in this app and what is not. How secure is it? I honestly don't know. And what part of my communication can be read by thee provider? Is there a zero-knowledge-solution?

Due to the fact that encrypting data is one of the most important things in the technological industry now, I'd be happy if there were a clear overview about the encryption and security in Rocket.Chat. A possible solution for that, is not publishing a whitepaper, but it's publishing a clear graphic with explanation on the Rocket.Chat website. We non-coders need to understand what we are subscribing for here :).

Would be nice if the Rocket.Chat team would publish that information in a user friendly way.

Yours

**Comment by rafaelks** [comment]
@GoetheG Thanks for the question. Is your question specific to the iOS application? Because the iOS app doesn't have end-to-end encryption support yet, so the only encrypted message that we have in the app right now is the SSL (required) in every network request the app makes, including the WebSocket communication.

**Comment by GoetheG** [comment]
Hi @rafaelks. I am using iOS too. but this was a general question. I'd like to understand how secure Rocket.Chat really is. What's the difference between Slack and Rocket.Chat regarding security? Slack can read the conversations, that's for sure. But if the chat is not encrypted in Rocket.Chat, then the Rocket.Chat hoster can read the messages as well. So what's the difference?

It would be nice if there were a website on your website dedicated to security and encryption :). That would make it easier to understand how this thing works.

Cheers.

**Comment by rafaelks** [comment]
@GoetheG Thanks for the suggestion! We're going to pass your idea to the website team so they can add it to their roadmap, I think it's a great idea to have a page or docs explaining the security of our clients.

As I said before, the only security layer that we have right now on the iOS and Android application is the SSL network layer. The database isn't encrypted because we don't support E2E on the mobile apps yet and some information on the web client also aren't encrypted even when you're using E2E, like the attachments: images, videos, voice messages, etc.

I'm going to close this issue for now if that's OK for you, I think we answered your question, and we already passed your suggestion to the responsible team.

Thank you!
